WalletWasabi.Gui/Tabs/SettingsViewModel.cs

```diff
@@ -387,72 +381,60 @@ private async Task OpenConfigFileAsync()
 
 		#region Validation
 
-		public ErrorDescriptors ValidateSomePrivacyLevel()
-			=> ValidatePrivacyLevel(SomePrivacyLevel, whiteSpaceOk: true);
+		private void ValidateSomePrivacyLevel(IValidationErrors errors)
+			=> ValidatePrivacyLevel(errors, SomePrivacyLevel, whiteSpaceOk: true);
 
-		public ErrorDescriptors ValidateFinePrivacyLevel()
-			=> ValidatePrivacyLevel(FinePrivacyLevel, whiteSpaceOk: true);
+		private void ValidateFinePrivacyLevel(IValidationErrors errors)
+			=> ValidatePrivacyLevel(errors, FinePrivacyLevel, whiteSpaceOk: true);
 
-		public ErrorDescriptors ValidateStrongPrivacyLevel()
-			=> ValidatePrivacyLevel(StrongPrivacyLevel, whiteSpaceOk: true);
+		private void ValidateStrongPrivacyLevel(IValidationErrors errors)
+			=> ValidatePrivacyLevel(errors, StrongPrivacyLevel, whiteSpaceOk: true);
 
-		public ErrorDescriptors ValidateDustThreshold()
-			=> ValidateDustThreshold(DustThreshold, whiteSpaceOk: true);
+		private void ValidateDustThreshold(IValidationErrors errors)
+			=> ValidateDustThreshold(errors, DustThreshold, whiteSpaceOk: true);
 
-		public ErrorDescriptors ValidateTorSocks5EndPoint()
-			=> ValidateEndPoint(TorSocks5EndPoint, Constants.DefaultTorSocksPort, whiteSpaceOk: true);
+		private void ValidateTorSocks5EndPoint(IValidationErrors errors)
+			=> ValidateEndPoint(errors, TorSocks5EndPoint, Constants.DefaultTorSocksPort, whiteSpaceOk: true);
 
-		public ErrorDescriptors ValidateBitcoinP2pEndPoint()
-			=> ValidateEndPoint(BitcoinP2pEndPoint, Network.DefaultPort, whiteSpaceOk: true);
+		private void ValidateBitcoinP2pEndPoint(IValidationErrors errors)
+			=> ValidateEndPoint(errors, BitcoinP2pEndPoint, Network.DefaultPort, whiteSpaceOk: true);
 
-		public ErrorDescriptors ValidatePrivacyLevel(string value, bool whiteSpaceOk)
+		private void ValidatePrivacyLevel(IValidationErrors errors, string value, bool whiteSpaceOk)
 		{
-			if (whiteSpaceOk && string.IsNullOrWhiteSpace(value))
-			{
-				return ErrorDescriptors.Empty;
-			}
-
-			if (uint.TryParse(value, out _))
+			if (!whiteSpaceOk || !string.IsNullOrWhiteSpace(value))
```

Comment:
Why did you change the format here?

Reply:
I inverted the if, to get rid of now redunant return statement..

otherwise it would look like:
```
if (whiteSpaceOk && string.IsNullOrWhiteSpace(dustThreshold))
			{
				return;
			}
			if (!string.IsNullOrEmpty(dustThreshold) && dustThreshold.Contains(',', StringComparison.InvariantCultureIgnoreCase))
			{
				errors.Add(ErrorSeverity.Error, "Use decimal point instead of comma.");
			}

			if (!decimal.TryParse(dustThreshold, out var dust) || dust < 0)
			{
				errors.Add(ErrorSeverity.Error, "Invalid dust threshold.");
			}
```

Reply:
to avoid redundant return statement, we no longer return stuff from validate methods.